locale/en_US/LC_MESSAGES/clr-installer.po

```diff
@@ -809,3 +809,23 @@ msgstr "Network Required"
 
 msgid "Installation Steps Complete"
 msgstr "Installation Steps Complete"
+
+#, c-format
+msgid "Password is too similar to the user attribute: %s"
+msgstr "Password is too similar to the user attribute: %s"
+
+msgid "Password should contain atleast one (special character, digit & Uppercase)"
+msgstr "Password should contain atleast one (special character, digit & Uppercase)"
+
+# cracklib messages
+msgid "it does not contain enough DIFFERENT characters"
+msgstr "Password does not contain enough DIFFERENT characters"
+
+msgid "it is too simplistic/systematic"
+msgstr "Password is too simplistic/systematic"
+
+msgid "it is based on a dictionary word"
+msgstr "Password is based on a dictionary word"
+
+msgid "it is based on a (reversed) dictionary word"
+msgstr "Password is based on a (reversed) dictionary word"
```

Comment:
Need a carriage return at the end of this line.

Reply:
Okay Thanks...Done and ensured in all PO files